dynamiq/memory/backends/pinecone.py

```diff
@@ -68,12 +72,30 @@ def model_post_init(self, __context) -> None:
         if not self.vector_store._index:
             raise PineconeError("Failed to initialize Pinecone index")
 
+        # Configure embedder truncation settings
+        if hasattr(self.embedder.document_embedder, "enable_truncation"):
```

Comment:
Such params were added to base embedder, so as far as I understand they always should be present